assets/script.js

```diff
@@ -0,0 +1,50 @@
+// Reminder: Ctrl + Click a url to open it in your browser
+// Dog API documentation link: https://api-ninjas.com/api/dogs
+var dog = "Dachshund"
+    $.ajax({
```

Comment:
This ajax call looks the same as below for dog 2, i think we can make a function?

Reply:
I will work on that asap, not really much time left today. This might be an endeavor for during class time tomorrow.